test/models/setting_test.rb

```diff
@@ -63,4 +63,13 @@ def test_class_methods_are_shorthand_for_values
 
     assert_equal(10, Setting.wishlist_size)
   end
+
+  def test_multilocal_terms_and_conditions
+    Setting.find_by(code: :terms_and_conditions_link).update(value: "{\"en\":\"https://example.com\", \"et\":\"https://example.et\"}")
```

Comment:
Fail fast (`update!`). Otherwise it fails silently if `update` does not succeed for some reason.

Reply:
Also, if you really need to supply JSON, why not doing it in a more straightforward fashion: `{ en: 'link' }`?

Reply:
Well, now here we are testing the saving/validation correctness as well as value parsing.
But deleted I18n-based tests, there is no sense in testing localisation anyway.